sally.yaml

```diff
@@ -25,3 +25,5 @@ packages:
         repo: github.com/yarpc/yarpc-go
     zap:
         repo: github.com/uber-go/zap
+    cadence:
```

Comment:
Ok. after running, i see this listed.

-- 
atomic - github.com/uber-go/atomic
cadence - github.com/uber-go/cadence-client